clang/runtime/dpct-rt/include/dpct/dpl_extras/dpcpp_extensions.h

```diff
@@ -566,6 +566,49 @@ class radix_sort {
   uint8_t *_local_memory;
 };
 
+/// Load linear segment items into block format across threads
+/// Helper for Block Load
+template <size_t GROUP_WORK_ITEMS, typename InputT, int ITEMS_PER_THREAD,
```

Comment:
I'd prefer we group all template value parameters together and all template type parameters together unless there is a compelling reason to do otherwise.

Reply:
Yes this is updated